### ISSUE TITLE
$wgIframe in LocalSettings.php ignored

### ISSUE THREAD

**Issue by McShelby** [sigbertklinke/Iframe#1]
Any server key addition in LocalSettings.php like

`$wgIframe['server']['example'] = array('scheme' => 'http', 'domain' => 'example.com');`

is ignored and can not be used by a later iframe 

`<iframe key="example" path="" />`

**Comment by sigbertklinke** [comment]
True, I talked with the Mediawiki poeple and the "old" and the "new" initialization algorithm working differently. Guess I will find a solution.

**Comment by johnyradio** [comment]
does this mean the extension is unusable now?

**Comment by sigbertklinke** [comment]
Uploaded a new version 0.06 which reads the global variable $wgIframe correctly.
